### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by eugeniykurasov** [jianyuan/terraform-provider-sentry#251] [General]
How often provider release new version?
From 0.10.0 till master already has changes with new features.

Thanks

**Comment by jianyuan** [comment]
I will release a new version shortly 👌

**Reply by jianyuan** [reply]
[v0.11.0](https://github.com/jianyuan/terraform-provider-sentry/releases/tag/v0.11.0) is out 🎉

**Comment by eugeniykurasov** [comment]
Do you can create new release? 
`0.11.1` fixed async create metric alerts. 
Thanks

**Reply by jianyuan** [reply]
Not yet. I'm trying to get the acceptance tests to pass.

**Reply by eugeniykurasov** [reply]
Do you need help?

**Reply by jianyuan** [reply]
@eugeniykurasov It seems to be related to the Sentry organization the tests are using. Their API is responding slowly, possibly because there's too much historical test data. I'll try to clear them.
